samples/infrastructure.yaml

```diff
@@ -0,0 +1,55 @@
+## Kubernetes
+
+infrastructure:
+  name: prod-platformdemo-k8scluster
+  id: prodplatformdemok8scluster
+  description: ""
+  tags: {}
+  org: default
+  project: Platform_Demo
+  uses: kubernetes-gke| kubernetes-eks | kubernetes-aks | kubernetes-direct
+  with:
+    connector: platformdemok8s
+    namespace: e2e-prod
+    release: release-<+INFRA_KEY>
+    simultaneous-deployment: false
+    
+## ECS
+
+infrastructure:
+  name: ECS
+  id: ECS
+  description: "ECS Deployment Infrastructure"
+  tags:
+    account:dev
+  org: default
+  project: Platform_Demo
+  uses: ecs
+  with:
+    connector: AWSSalesDanF
+    region: us-west-2
+    cluster: lg-fargate
+    simultaneous-deployment: false
```

Comment:
would `parallel-deploy: true` be the same meaning as `simultaneous-deployment`?

Reply:
yes parallel deploy would work. Simultaneous deploy means multiple pipelines in parallel can deploy to the same infra def. We won't hold the subsequent pipelines, they will all run and risk overwriting each other.